### DISCUSSION TITLE
Can't get SIP to disable on my Mac. Help

### DISCUSSION THREAD

**Discussion by wisetv19** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#161] [Q&A]
How to get Sip to disable on my Mac. I't re-enable's when I cut it back on from safe mode.

**Comment by ycc06516** [comment]
boot into recovery -> Utilities -> Terminal -> csrutil disabled -> restart

**Reply by ASC-David** [reply]
This happens to me as well. MacPro5,1 running Monterey 12.2.1 installed using Dortania's Legacy Patcher, GUI version. When I boot in recovery, I can turn off SIP, but when I reboot out of recovery and try and install the Wireless USB OC Big Sur Adapter-V12 2, I get a "SIP id enabled" message instructing me to boot in recovery and disable SIP, etc. I think I need to disable SIP in OC, but I don't know how to do that. Ideas/next steps please.
